ci/matrix.xml

```diff
@@ -205,6 +206,8 @@
         ir/mimaReportBinaryIssues tools/mimaReportBinaryIssues \
         jsEnvs/mimaReportBinaryIssues testAdapter/mimaReportBinaryIssues \
         stubs/mimaReportBinaryIssues cli/mimaReportBinaryIssues
+    sbt ++$scala ir/compile:doc tools/compile:doc jsEnvs/compile:doc \
+        testAdapter/compile:doc
```

Comment:
Arrgh... I forgot to mention `stubs`... I didn't mean to have you remove it. Sorry, my bad